src/main/java/core/basesyntax/impl/FruitsCalculatorImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.database.DataBase;
+import core.basesyntax.service.FruitsCalculator;
+import java.util.List;
+import java.util.Map;
+
+public class FruitsCalculatorImpl implements FruitsCalculator {

```

Comment:
this class needs big refactoring
1. logic of parsing string line into a model (fruit activity) should be separated
2. indexes should be constants
3. You should create an enum for operation types
4. don't use switch, that's how how we should implement strategy pattern, which this class aspires to be, use HashMap, operation <-> handler